sky/provision/runpod/utils.py

```diff
@@ -27,11 +27,14 @@
     'RTX4070Ti': 'NVIDIA GeForce RTX 4070 Ti',
     'RTX4080': 'NVIDIA GeForce RTX 4080',
     'RTX4090': 'NVIDIA GeForce RTX 4090',
-    'H100-80GB-HBM3': 'NVIDIA H100 80GB HBM3',
-    'H100-PCIe': 'NVIDIA H100 PCIe',
+    # Following instance is displayed as SXM at the console
+    # but the ID from the API appears as HBM
+    'H100-80GB-SXM': 'NVIDIA H100 80GB HBM3',
```

Comment:
Are H100 GPUs always have 80GB memory? Should we just call it `H100-SXM`?

Reply:
Updated!